lib/dor/services/client.rb

```diff
@@ -153,6 +153,7 @@ def build_connection(with_retries: false)
           #   the Faraday instance will be passed an empty block, which
           #   causes the adapter not to be set. Thus, everything breaks.
           builder.adapter Faraday.default_adapter
+          builder.options[:timeout] = 600 # 10 minutes timeout to wait for a response
```

Comment:
going from 60 seconds to 600 seconds seems like a big jump;  I might go with 300 seconds.

Reply:
OK, i reduced to 300 seconds.